.github/workflows/release.yml

```
@@ -114,3 +114,23 @@ jobs:
           asset_path: ./ittapi_build_${{ github.ref_name }}.zip
           asset_name: ittapi_build_${{ github.ref_name }}.zip
           asset_content_type: application/zip
+  
+  # upload the built wheels to pypi
+  upload_to_pypi:
+    name: Upload the wheel files to pypi
+    runs-on: ubuntu-latest
+    needs: [python_build, create_release]
+    steps:
+      - name: Download artifacts
+        uses: actions/download-artifact@6b208ae046db98c579e8a3aa621ab581ff575935 # v4.1.1
+        with:
+          pattern: build-artifacts-*
+          merge-multiple: true
+      - name: List wheel files
+        run: find -name *.whl
+      - name: Publish the package to pypi
+        uses: pypa/gh-action-pypi-publish@release/v1
```

```suggestion
        uses: pypa/gh-action-pypi-publish@0ab0b79471669eb3a4d647e625009c62f9f3b241 # v1.10.1
```
Using branch pointers is not a safe solution.
Instead of using branch pointers it is recommended to use sha1 commit identifiers. This will make your workflows more secure and better reproducible, saving you from sudden and unpleasant surprises.
https://julienrenaux.fr/2019/12/20/github-actions-security-risk/

---

I have made the change in the recent commit, please have a look.